eng/scripts/Setup-MQTTContainer.ps1

```diff
@@ -0,0 +1,24 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $AgentImage,
+    [string] $ConfigFile
+)
+
+if ($IsLinux -and $AgentImage -match "ubuntu") {
+    docker pull eclipse-mosquitto
+
+    sudo docker run -d -p 1883:1883 -p 9001:9001 -v ${$ConfigFile}:/mosquitto/config/mosquitto.conf eclipse-mosquitto
+
+    # Testing
+
+    sudo apt install net-tools
```

Comment:
Install `net-tools` using `AptDependencies` in the matrix. If a row for `AptDependencies` is not in the matrix go ahead and create it.

Reply:
This was to debug the connection, I have removed those installs